azure-pipelines.yml

```diff
@@ -0,0 +1,34 @@
+pool:
```

Comment:
will this run as part of the PR merge (assuming this also runs unit tests)?

Reply:
Yes, please see https://dev.azure.com/LogicApps-connector-extensions/builtInConnectorExtensibility/_build/results?buildId=12&view=ms.vss-test-web.build-test-results-tab